<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="INHYDSRIKANTHP" name="com.moveware.system.codes.ManageCodesNonUI" tests="1" failures="1" timestamp="01 Jul 2019 05:52:02 GMT" time="27.308" errors="0">
  <testcase name="addCodesNumber" time="27.308" classname="com.moveware.system.codes.ManageCodesNonUI">
    <failure type="java.lang.AssertionError" message="Record Not Saved">
      <![CDATA[java.lang.AssertionError: Record Not Saved
at org.testng.Assert.fail(Assert.java:96)
at com.moveware.objectRepository.SystemCodesNonUIObjects.codesTotalRecords(SystemCodesNonUIObjects.java:628)
at com.moveware.objectRepository.SystemCodesNonUIObjects.enterCodesDatainOfCodeTypeNumber(SystemCodesNonUIObjects.java:577)
at com.moveware.objectRepository.SystemCodesNonUIObjects.executeFunctionalitySequence(SystemCodesNonUIObjects.java:260)
at com.moveware.system.codes.ManageCodesNonUI.addCodesNumber(ManageCodesNonUI.java:88)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- addCodesNumber -->
</testsuite> <!-- com.moveware.system.codes.ManageCodesNonUI -->
